acceptance/openstack/loadbalancer/v2/loadbalancers_test.go

```diff
@@ -134,6 +134,18 @@ func TestLoadbalancersCRUD(t *testing.T) {
 		t.Fatalf("Unable to create l7 rule: %v", err)
 	}
 
+	allPages, err = l7policies.ListRules(lbClient, policy.ID, l7policies.ListRulesOpts{}).AllPages()
+	if err != nil {
+		t.Fatalf("Unable to get l7 rules: %v", err)
+	}
+	allRules, err = l7policies.ExtractRules(allPages)
```

Comment:
Need `:=` here instead of `=`

Reply:
Thanks @jtopjian , I will fix this in the following PR. BTW, do you know if there is anyone working on supporting Octavia in openlab environment?

Reply:
I've opened an issue here to discuss:

https://github.com/theopenlab/openlab-zuul-jobs/issues/143

Hopefully we'll have it enabled soon :)